app/controllers/home_controller.rb

```diff
@@ -0,0 +1,4 @@
+class HomeController < ApplicationController
+  def index; end
+end
+
```

Comment:
Remove this extra blank space.

Also why do we need this controller?
Our backend will have apis only so we don;t need this UI part in rails.

Reply:
Completed and removed devise views too, only rails api